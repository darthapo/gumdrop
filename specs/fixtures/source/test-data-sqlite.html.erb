<ul>
<% data.sql_data.posts.each do |item| %>
  <li><%= item.title %></li>
<% end %>
</ul>
