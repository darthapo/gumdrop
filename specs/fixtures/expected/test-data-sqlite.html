<body><ul>
  <li>Test Title</li>
</ul>
</body>